pkg/network/ebpf/c/protocols/http2/decoding.h

```diff
@@ -326,6 +326,15 @@ static __always_inline void parse_frame(struct __sk_buff *skb, skb_info_t *skb_i
         skb_info->data_off += current_frame->length;
     }
 
+    // When we accept an RST with NO ERROR, it means that the current stream was terminated with no error.
+    if (current_frame->type == kRSTStreamFrame) {
+        __u32 rst_error_code = 0;
+        bpf_skb_load_bytes(skb, skb_info->data_off, &rst_error_code, HTTP2_RST_ERROR_SIZE);
+        if (rst_error_code == HTTP2_RST_NO_ERROR) {
+            handle_end_of_stream(current_stream, &http2_ctx->http2_stream_key);
+        }
+    }
```

Comment:
it is better to declare a variable (i.e. `handle`)
and 
```
end_of_stream = is_rst || (current_frame->flags & HTTP2_END_OF_STREAM) == HTTP2_END_OF_STREAM);

if (end_of_stream) ....
```

Why is it better?
We're using static inline functions, meaning we're copying the functions during compilation, and by that we increase the binary size / the number of branches the verifier has to scan, which can push us over the limit (since those programs already close to the limit)